weave/scorers/bedrock_guardrails.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any
+
+from pydantic import Field, PrivateAttr
+
+import weave
+from weave.flow.scorer import WeaveScorerResult
+from weave.integrations.bedrock import patch_client
+
+
+class BedrockGuardrailScorer(weave.Scorer):
+    """
+    The `BedrockGuardrailScorer` class is a guardrail that leverages AWS Bedrock's
+    guardrail functionality to detect and filter content based on configured policies.
+
+    Attributes:
+        guardrail_id (str): The identifier of the guardrail to use.
+        guardrail_version (str): The version of the guardrail to use.
+        source (str): The source of the content to evaluate, either 'INPUT' or 'OUTPUT'.
+
+    Note:
+        This scorer requires AWS Bedrock client to be properly configured with
+        appropriate permissions to access the guardrail API.
+    """
+
+    guardrail_id: str = Field(description="The identifier of the guardrail to use.")
+    guardrail_version: str = Field(
```

Comment:
What is this?

Reply:
terrible boto3 api

Reply:
But what does it mean?  I don't understand what this setting does